### DISCUSSION TITLE
Are keywords (for blocking) case sensitive?

### DISCUSSION THREAD

**Discussion by ghost** [FossifyOrg/General-Discussion#504] [Q&A] [answered]
Are keywords (for blocking) case sensitive?

**Comment by naveensingh** [comment, marked as answer]
No. 

https://github.com/FossifyOrg/Messages/blob/d6b3d3b1227893a5619f6ae7c852dfe5f82e2106/app/src/main/kotlin/org/fossify/messages/receivers/SmsReceiver.kt#L126-L135

**Reply by ghost** [reply]
Thanks @naveensingh for such a prompt reply! Just what I needed. If you don't mind then I have more questions for you. 

2) Does blocking by keyword support other languages and not just English? 
3) Does it only filter by body or also look into the sender name? It seems to be looking only for body content from what I was able to make out from above code but not sure.

**Reply by naveensingh** [reply]
> Does blocking by keyword support other languages and not just English?

Yes, generally, not sure about special cases.

> It seems to be looking only for body content from what I was able to make out from above code but not sure.

Your insight serves you well.
